sdk/go/common/apitype/property-values.json

```diff
@@ -0,0 +1,232 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/pulumi/pulumi/blob/master/sdk/go/common/apitype/property-values.json",
+    "$anchor": "property_value",
+    "title": "Pulumi Property Value",
+    "description": "A schema for Pulumi Property values.",
```

Comment:
OK this schema sounds extremely important in here, that is the piece of knowledge I really wanted to have formalized, props..